spec/gears/table_spec.lua

```diff
@@ -42,6 +42,20 @@ describe("gears.table", function()
         end)
     end)
 
+
+    describe("table.iterate_value", function()
+        it("nil argument", function()
+            local t = { "a", "b", "c", "d" }
+            local f = gtable.iterate_value(t, "a")
+            assert.is.same(f, "b")
+        end)
+        it("with step size", function()
+            local t = { "a", "b", "c", "d" }
+            local f = gtable.iterate_value(t, "a", 2)
+            assert.is.same(f, "c")
+        end)
+    end)
+
```

Comment:
i think it would be useful to add one more testcase using a `filter` argument, to make the coverage bot happy

Reply:
Yeah, I can add that